x-pack/plugins/security_solution/common/api/detection_engine/prebuilt_rules/get_prebuilt_rules_and_timelines_status/get_prebuilt_rules_and_timelines_status_route.schema.yaml

```diff
@@ -5,8 +5,9 @@ info:
 paths:
   /api/detection_engine/rules/prepackaged/_status:
     get:
-      operationId: GetPrebuiltRulesAndTimelinesStatus
+      x-labels: [ess]
       x-codegen-enabled: true
+      operationId: GetPrebuiltRulesAndTimelinesStatus
```

Comment:
Is it because we're going to disable the legacy `/api/detection_engine/rules/prepackaged` endpoints in Serverless?

I don't think we have a ticket to todo list item for that, or even ever discussed this.

Reply:
Yes. In general we discussed that deprecated API endpoints shouldn't be available in Serverless. It's not used by UI and is a good candidate to be excluded in Serverless.